Assets/BossRoom/Scripts/Client/UI/MainMenuUI.cs

```diff
@@ -178,6 +178,9 @@ private void ConnectStatusToMessage(ConnectStatus status, bool connecting)
                     var text = connecting ? "Something went wrong" : "The connection to the host was lost";
                     m_ResponsePopup.SetupNotifierDisplay(title, text, false, true);
                     break;
+                case ConnectStatus.IncompatibleBuildType:
+                    m_ResponsePopup.SetupNotifierDisplay("Connection Failed", "Server and client builds are not compatible. You cannot connect a release build to a development build or an in-editor session", false, true);

```

Comment:
Just a comment to add a trailing period to this and to the other messages for uniformity.

Reply:
I think the case for `ConnectStatus.GenericDisconnect` needs a punctuation pass. Approval once done.

Reply:
Done